src/pubsub/ua_pubsub_securitygroup.c

```diff
@@ -108,7 +108,7 @@ updateSKSKeyStorage(UA_Server *server, UA_SecurityGroup *securityGroup){
         /* the first item has the oldest key */
         UA_PubSubKeyListItem *oldestKey = TAILQ_FIRST(&keyStorage->keyList);
         /* shift head to second oldest */
-        TAILQ_INSERT_HEAD(&keyStorage->keyList, oldestKey->keyListEntry.tqe_next, keyListEntry);
```

Comment:
You can just put an overall comment ‘move oldest to the tail’.
Some of the line-comments no longer apply.
I.e. shift head to second oldest.